src/components/DrawingCursor.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { connect } from '../store/Connect';
+import '../../sass/components/_drawing-cursor.scss';
+
+const DrawingCursor = ({
+    display,
+    left,
+    top,
+}) => (
+    <div className="drawing-mouse-pointer" style={{ top, left, display }} />
```

Comment:
I think we would want the pencil icon to also change with theme? You'll need to use `Icons.jsx` to do this instead of defining the background image in CSS

Reply:
![image](https://user-images.githubusercontent.com/3080372/44331422-ad972b00-a47e-11e8-9cdb-65ad08c916a5.png)
![image](https://user-images.githubusercontent.com/3080372/44331440-b7209300-a47e-11e8-9d5c-92e657296729.png)

Reply:
ah. It's a pass then